pyvista/utilities/parametric_objects.py

```diff
@@ -1003,6 +1008,11 @@ def ParametricSuperEllipsoid(xradius=None, yradius=None, zradius=None,
     pyvista.PolyData
         ParametricSuperEllipsoid surface.
 
+    See Also
+    --------
+    ParametricSuperToroid : Toroidal equivalent of ParametricSuperEllipsoid.
```

Comment:
```suggestion
    pyvista.ParametricSuperToroid : Toroidal equivalent of ParametricSuperEllipsoid.
```

Reply:
![image](https://user-images.githubusercontent.com/39341281/143302168-b561023c-1ea7-436a-8f94-bccf2ae3247b.png)

After fixing these issues, the links work either way.  But I think we should be consistent.  I'd use `pyvista.X` here to highlight that it is available in the `pyvista` namespace.

Reply:
I was following other examples of "See Also" which reference functions in the local sub-module, but those are probably not available in the `pyvista` namespace so this makes sense.